models/src/test/scala/coop/rchain/models/RhoTypesTest.scala

```diff
@@ -44,7 +45,7 @@ class RhoTypesTest extends FlatSpec with PropertyChecks with Matchers {
     })
   }
 
-  def roundTripSerialization[A: Serialize: Arbitrary: Pretty](implicit tag: ClassTag[A]): Unit =
+  def roundTripSerialization[A: Serialize: Arbitrary: Shrink: Pretty](implicit tag: ClassTag[A]): Unit =
```

Comment:
Oh, I would have thought that shapeless does it automagically.

Reply:
It does, but only in testImplicits. Which is good, b/c it allows for caching the derived instances instead of deriving them anew in every place we'd be importing ScalacheckShapeless._ otherwise